frontend/src/components/Learn Lessons/C1L2.tsx

```diff
@@ -0,0 +1,60 @@
+export default function C1L2({ children }) {
+  return (
+    <div className={`normalLesson ${open && "showing"}`} style={{ color: "white" }}>
+      <h3 style={{ color: "white", textAlign: "left", marginLeft: "-3%", marginRight: "-3%" }}>
+        {" "}

```

Comment:
What is the purpose of all the single space jsx blocks `{" "}`? They don't end up rendering in the DOM.